components/site/tests/site.rs

```diff
@@ -298,7 +298,7 @@ fn can_build_site_with_taxonomies() {
     assert!(file_contains!(
         public,
         "categories/a/rss.xml",
-        "https://replace-this-with-your-url.com/categories/a/rss.xml"
+        "https://replace-this-with-your-url.com/posts/with-assets/"
```

Comment:
The feed no longer contains a link to itself. Assert that the category feed contains a link to one of its posts instead.